cmd/podman/healthcheck_run.go

```diff
@@ -43,12 +43,11 @@ func healthCheckCmd(c *cliconfig.HealthCheckValues) error {
 		return errors.Wrap(err, "could not get runtime")
 	}
 	status, err := runtime.HealthCheck(c)
-	if err != nil {
-		if status == libpod.HealthCheckFailure {
-			fmt.Println("\nunhealthy")
-		}
-		return err
+	if status == libpod.HealthCheckFailure {
```

Comment:
What're these changes? Looks like we're discarding health check errors?

Reply:
they were because the exitCode wasn't always being set if a failure happened (as c.Exec now returns the exit code it receieved, as opposed to just an error, so a failed check would have a nil error but would have actually errored). Though, I did change the handling of this to libpod/healthcheck.go instead of here